doc/admin/conf_files/krb5_conf.rst

```diff
@@ -515,10 +515,10 @@ following tags may be specified in the realm's subsection:
     host will be tried.
 
 **master_kdc**
-    Identifies the master KDC(s).  Currently, this tag is used in only
+    Identifies the primary KDC(s).  Currently, this tag is used in only
```

Comment:
This seems like it's setting us up for admin confusion.  Our docs change to talk only about primary, but there's no `primary_kdc` parameter.  It seems like it'd be more clear to either rename the parameter (and keep backward compatibility) if this change is desired.

There's some precedent for this - for instance, `iprop_replica_poll` in 07d9f95e433cf59674faedf1121fe810ab3683c7 as part of the rename in #851.

Reply:
(This has been resolved, though I'm unable to mark it as such for some reason.)